README.md

```diff
@@ -347,19 +347,15 @@ You can observe these in action on this screenshot:
 
 Prerequisites:
 
-- The Go tool chain
-- `dep` for Go dependency management
 - `make`
 - Docker
 
 To build from source:
 
 ```console
-$ dep ensure         # to install dependencies into vendor/
 $ make lint          # to run linters
 $ make test          # to run tests
-$ make build         # to build local binaries
-$ make docker-build  # to build Docker images
+$ make build         # to run multi-stage Docker build of binaries and images
```

Comment:
Do we want a `make push` target to add to the list here (and in the Makefile, of course)?  (Perhaps showing how to push to personal repos, e.g. `DOCKER_REGISTRY=myrepo make push`?  (Or reference the pre-existing `push-all-images` target)

Reply:
Sure. On it.